pcdsdevices/epics_motor.py

```diff
@@ -125,6 +127,31 @@ def limits(self, limits):
         self.low_limit = limits[0]
         self.high_limit = limits[1]
 
+    @property
+    def motor_status(self):
```

Comment:
This is a little redundant. If you make the signal we can just call -> `disabled.get()` or `disabled.set()`. I would scratch these these two properties and just make a function that looks like:

```python
def enable(self, enable):
    return self.disabled.set(not enable)
```